HelpSource/Classes/Document.schelp

```diff
@@ -239,6 +239,20 @@ Task({
 )
 ::
 
+method:: save
```

Comment:
An open question: do you think it would be worthwhile to also link to this method right below the `Document.open` entry? It's really far down from the `open` class method and the top of the file, and one could argue they are related (even though the first one is class method, the other an instance method). OTOH it can be found if you search for `save` or trigger help for the method name in the first place... so maybe that's not needed? This came to me when I was scrolling through the help file trying to find the method manually.

Reply:
Don't mind. Can just move it if that makes more sense?

Reply:
It's tricky! `save` is an instance method, while `open` is a class method... So while I'd like to see them together, this would be against the existing convention of separating class and instance methods. That's why I think a link would be better.

Reply:
Done. Also did a link from save back to *open.